site/content/en/docs/build-test-update.md

```diff
@@ -142,11 +142,6 @@ export KUBECONFIG='/<path to user dir>/.kube/kind-config-mkpod'
 ```
 After pointing to the correct master you will be able to drop into the container using `kubectl exec -it <pod name> <bash/sh/etc>`. **This pod will only last the lifecycle of the job, if you need more time to debug you might add a `sleep` within the job execution.
 
-#### Using Phaino
-[Phaino](/docs/components/cli-tools/phaino/) lets you interactively mock and run the job locally on
-your workstation in a docker container. Detailed instructions can be found in
-Phaino's [Readme](/docs/components/cli-tools/phaino/).
-
 Note: Test containers designed for decorated jobs (configured with `decorate: true`)
```

Comment:
I am not sure whether we should delete this paragraph (L145-149) too, or not.

```diff

##### Debugging within a pj-on-kind.sh container
To point `kubectl` to the Kind cluster you will need to export the `KUBECONFIG` Env. The command to point this to the correct config is echoed in the pj-on-kind.sh logging. It will have the form:
(SOME CODEBLOCK)
After pointing to the correct master you will be able to drop into the container using `kubectl exec -it <pod name> <bash/sh/etc>`. **This pod will only last the lifecycle of the job, if you need more time to debug you might add a `sleep` within the job execution.

-Note: Test containers designed for decorated jobs (configured with `decorate: true`)
-may behave incorrectly or fail entirely without the environment the pod utilities
-provide. Similarly jobs that mount volumes or use `extra_refs` likely won't work
-properly.
-These jobs are best run locally as decorated pods inside a [Kind] cluster [Using pj-on-kind.sh](#using-pj-on-kindsh).

### How to manually run a given job on Prow

```

Any comment would be appreciated.

Reply:
Yes, this is part of the "Using Phaino" section and only applies to Phaino. Please remove this as well.